app/components/Link/Link.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { Text, Linking } from 'react-native'
+
+import styles from "./style"
+
+const Link = ({ url }) => {
```

Comment:
It would be great to have second param text in case if we don't want to display a very long URL.
And if text is not provided, show url value instead: {text || url}

Reply:
done